npm/angular/tsconfig.app.json

```diff
@@ -6,14 +6,11 @@
   },
   "files": [
     "src/main.ts",
-    "src/polyfills.ts",
-    "node_modules/cypress/types/cypress-global-vars.d.ts"
-  ],
-  "include": [
-    "src/**/*.ts"
```

Comment:
there is no reason to use this `include` as the `main.ts` and `polyfills.ts` file should serve as the only entry points into the app